x-pack/filebeat/input/httpjson/chain.go

```diff
@@ -93,3 +93,26 @@ type responseChainConfig struct {
 	Transforms transformsConfig `config:"transforms"`
 	Split      *splitConfig     `config:"split"`
 }
+
+func defaultChainConfig() config {
+	chaincfg := defaultConfig()
+	chaincfg.Chain = []chainConfig{
+		{
+			While: &whileConfig{
+				Auth:     chaincfg.Auth,
+				Request:  *chaincfg.Request,
+				Response: responseChainConfig{},
+				Replace:  "",
+				Till:     nil,
```

Comment:
These zero values can be omitted unless you think having them there provides information to the reader.

Reply:
sure , will update these